org.openstreetmap.josm.appdata.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Has the file been upstreamed?

Reply:
yes.
I have included this file, because upstream does not want to maintain the releases in the appstream file.